src/commands/overrides/get.ts

```diff
@@ -0,0 +1,92 @@
+import { Flags } from '@oclif/core'
+import inquirer from '../../ui/autocomplete'
+import {
+    environmentPrompt, 
+    EnvironmentPromptResult, 
+    featurePrompt, 
+    FeaturePromptResult 
+} from '../../ui/prompts'
+import Base from '../base'
+import { fetchFeatureOverridesForUser } from '../../api/overrides'
+import { fetchEnvironmentByKey } from '../../api/environments'
+import { fetchVariationByKey } from '../../api/variations'
+
+export default class DetailedTargeting extends Base {
```

Comment:
should this be named something else?
```suggestion
export default class DetailedOverrides extends Base {
```